### ISSUE TITLE
COMMENT ON unqualified  table: ERROR:  relation "title_memorial" does not exist

### ISSUE THREAD

**Issue by strk** [linz/linz-lds-bde-schema#100]
The generated `sql/08-lds_comments.sql` file contains statements defining comments for unqualified table names (without schema). This result in this problem upon running the schema loader twice:
```
$ linz-lds-bde-schema-load linz-lds-bde-schema-test-db
Loading /usr/share/linz-bde-uploader/sql//01-bde_control_tables.sql
Loading /usr/share/linz-bde-uploader/sql//02-bde_control_functions.sql
Loading /usr/share/linz-bde-uploader/sql//03-bde_control_version.sql
Loading /usr/share/linz-bde-uploader/sql//99-patches.sql
SET
SET
CREATE EXTENSION
DO
DO
DO
INFO:  Patch BDE - 1.0.2: Create survey tables and columns for pending parcels release is already applied
INFO:  Patch lds_bde - 1.2.0: Create completion-date equipped Title Instrument table is already applied
DO
BEGIN
DO
CREATE FUNCTION
...
COMMENT
COMMENT
COMMENT
ERROR:  relation "title_memorial" does not exist
```

**Comment by strk** [comment]
The problem was likely introduced by 3fcfbee640d9c5a1c1422fd27a34b42bc8cbef12, which stops modifying the "search_path" as part of loading the SQL files

**Comment by strk** [comment]
@Ashleigh234 can you comment on why some tables are qualified and other are not, in that generated file ?

**Comment by strk** [comment]
Ok it was an hard-coded mapping in comment extraction script, fixed with commit bd36e8e in PR #99
